tesseract_process_managers/src/task_generators/fix_state_bounds_task_generator.cpp

```diff
@@ -85,6 +85,8 @@ int FixStateBoundsTaskGenerator::conditionalProcess(TaskInput input, std::size_t
   }
 
   auto limits = fwd_kin->getLimits().joint_limits;
+  limits.col(0) = limits.col(0).array() + cur_composite_profile->lower_bounds_reduction;
```

Comment:
Should it only adjust those that are outside the bounds?

Reply:
This is reducing the bounds by a small amount. The checking whether or not it is outside of the bounds happens below

Reply:
I rebased and changed this